src/libstd/sys/vxworks/thread.rs

```diff
@@ -77,7 +77,7 @@ impl Thread {
     }
 
     pub fn set_name(_name: &CStr) {
-        assert!(false, "FIXME: set_name");
+        // do nothing
```

Comment:
Improve comment:

`// VxWorks does not provide a way to set the task name except at creation time`